packages/select/src/components/multi-select/_multi-select.scss

```diff
@@ -3,12 +3,25 @@
 
 @import "../../common/variables";
 
+$select-padding: $pt-grid-size * 0.5;
+
 .#{$ns}-multi-select {
   min-width: $pt-grid-size * 15;
 }
 
 .#{$ns}-multi-select-popover {
+  .#{$ns}-popover-content {
+    // use padding on container rather than margin on input group
+    // because top margin leaves some empty space with no background color.
+    padding: $select-padding;
+  }
+
   .#{$ns}-menu {
+    // offset the padding added by the multiselect component so menu can remain full width with padding on either side
+    // this is important so menu dividers can be full width, and so that menu items can have padding to the left and right
+    // for menu items
+    margin-left: -$select-padding;
+    margin-right: -$select-padding;
```

Comment:
Confirmed this doesn't break anything with existing MenuDividers

#### Current State (without the negative margins and without popover-content padding):
![Screenshot 2024-07-17 at 4 57 16 PM](https://github.com/user-attachments/assets/7385db4d-1bc2-4aa2-bcd8-0f818a502250)

#### New State (with the negative margins and with popover-content padding):
![Screenshot 2024-07-17 at 4 56 38 PM](https://github.com/user-attachments/assets/2536de8a-8e80-4924-9220-77ef04ac3a90)

Reply:
leaving note for followup, this appears in Select as well - would rather just keep this consistent with Select for now and not try to change both in this PR

---

note 5px above and below scrollbar

this PR
![Screenshot 2024-07-18 at 10 15 54 AM](https://github.com/user-attachments/assets/26182fcd-074d-41a1-8742-0aa84845c24d)

current:
![Screenshot 2024-07-18 at 10 15 50 AM](https://github.com/user-attachments/assets/8467713e-0e1b-447d-b5ce-9a5ba21e104e)